### DISCUSSION TITLE
Link Cloaking

### DISCUSSION THREAD

**Discussion by dedicatedcloud** [YOURLS/YOURLS#3737] [Q&A]
I know it's been asked before, and it says read the comments, but I can't find any.  So is Link Cloaking allowed or is it something that needs configuring at the registrar?

**Comment by dgw** [comment]
I see #477, #818, and #845 (and indeed, I wonder how we are supposed to read the comments on those Google Code issues now that the GC issue link redirects right back to GitHub).

This is not a feature built into YOURLS, no. It's likely possible to get a satisfactory result by modifying the [Toolbar sample plugin](https://github.com/YOURLS/YOURLS/tree/435b0804971031f412eb241e4a20402519a06501/user/plugins/sample-toolbar) included with YOURLS, possibly also combining the functionality of [Conditional Toolbar](https://github.com/YOURLS/conditional-toolbar) if you want it.
